plugins/scaffolder-backend/api-report.md

```diff
@@ -721,6 +736,10 @@ export type TaskCompletionState = 'failed' | 'completed';
 
 // @public
 export interface TaskContext {
+  // (undocumented)
+  abortSignal: AbortSignal;
+  // (undocumented)
+  cancelled: boolean;
```

Comment:
you can just do `.abortSignal.aborted` instead right, so we keep the information in one place